Source/WebGPU/WebGPU/ComputePipeline.h

```diff
@@ -73,18 +73,15 @@ class ComputePipeline : public WGPUComputePipelineImpl, public RefCounted<Comput
 
 private:
     ComputePipeline(id<MTLComputePipelineState>, const PipelineLayout&, MTLSize, Device&);
-    ComputePipeline(id<MTLComputePipelineState>, MTLComputePipelineReflection*, MTLSize, Device&);
+    ComputePipeline(id<MTLComputePipelineState>, Ref<PipelineLayout>, MTLSize, Device&);
```

Comment:
Can we give names to the second argument of these overloads? So it's clear that one is used for auto layout.

Reply:
`check-webkit-style` does not like it, unless you think I should ignore it:
```
ERROR: Source/WebGPU/WebGPU/ComputePipeline.h:75:  The parameter name "layout" adds no information, so it should be removed.
```

Reply:
oh, I meant something more descriptive than just `layout`.

```
ComputePipeline(id<MTLComputePipelineState>, const PipelineLayout& explicitLayout, MTLSize, Device&);
ComputePipeline(id<MTLComputePipelineState>, Ref<PipelineLayout> autoLayout, MTLSize, Device&);
```

Reply:
Had no idea that static analysis warning could be silenced by changing the name. I always thought it had to do with the parameter type - thank you!